docs/change-log.md

```diff
@@ -2,6 +2,21 @@
 
 This document provides an overview over the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 22nd, 2021
+
+Here is our first product update 2021. We started with some important new features into the new year.
+
+- **New:** We created an Environment Settings Screen that bundles all environment-specific settings. We moved the *Automation Rules* and environment-specific *Shared values and secrets overrides* into this new screen.  
+![Environment Confguration Screen](_assets/images/2021-01-22_Environment_Settings_Screen.png)
```

Comment:
Since it's shown on the screenshot, we could also mention that we allow now user to specify the Kubernetes namespace the environment will be deployed to. This has been a feature on our API already, but it also made it now to the UI.

Reply:
Done.